src/components/App.tsx

```diff
@@ -0,0 +1,29 @@
+import { Route, Switch } from 'wouter';
+import Landing from 'ribbit/components/Landing';
+import { useAuthStore } from 'ribbit/state/auth';
+import React from 'react';
+import supabase from 'ribbit/supabase';
+import Header from 'ribbit/components/Header';
+
+export default function App() {
+  // Initialize auth state
+  const setCurrentUser = useAuthStore((state) => state.setCurrentUser);
+  React.useEffect(() => {
+    async function initAuth() {
+      const {
+        data: { user },
+      } = await supabase.auth.getUser();
+      setCurrentUser(user);
+    }
+    initAuth();
+  }, [setCurrentUser]);
+
+  return (
+    <div id="app-container">
+      <Header />
+      <Switch>
+        <Route component={Landing} />
```

Comment:
I think eventually we'll want the default to be a 404 page, and just have `"/"` be the landing, right?